Meta/gn/secondary/AK/BUILD.gn

```diff
@@ -246,6 +246,7 @@ write_cmake_config("ak_debug_gen") {
     "LEXER_DEBUG=",
     "LIBWEB_CSS_ANIMATION_DEBUG=",
     "LIBWEB_CSS_DEBUG=",
+    "LIBWEB_WASM_DEBUG=",
```

Comment:
this change misses CSS_TRANSITIONS_DEBUG, perhaps a rebase on upstream is missing?

Reply:
Yeah, likely was based on an outdated checkout. Added.